book/src/plumbing/jars_and_ingredients.md

```diff
@@ -0,0 +1,231 @@
+# Jars and ingredients
+
+{{#include caveat.md}}
```

Comment:
Should the caveat be included here, as it is already in `plumbing.md`? If it should, the path isn't relative and it isn't showing up correctly.
```suggestion
{{#include ../caveat.md}}
```